server/src/utils.ts

```diff
@@ -54,15 +48,13 @@ export const getIpFromRequest = function(req: express.Request): string | null {
 }
 
 export const getHostNameFromRequest = async function(req: express.Request): Promise<string> {
-	// Get hostname from request
-	let hostname = req.params.hostname ?? 'unknown';
+	let hostname = req.hostname ?? 'unknown';
```

Comment:
Why did you remove the `.params` from this line? `hostname` should be retrieved from the request's URL parameters here, and this breaks that feature.

Reply:
That was an accident.